experimental/packages/otlp-exporter-base/src/platform/node/OTLPExporterNodeBase.ts

```diff
@@ -100,16 +99,18 @@ export abstract class OTLPExporterNodeBase<
       return;
     }
 
-    const promise = this._transport.send(data).then(response => {
-      if (response.status === 'success') {
-        onSuccess();
-        return;
-      }
-      if (response.status === 'failure' && response.error) {
-        onError(response.error);
-      }
-      onError(new OTLPExporterError('Export failed with unknown error'));
-    }, onError);
+    const promise = this._transport
+      .send(data, this.timeoutMillis)
+      .then(response => {
+        if (response.status === 'success') {
+          onSuccess();
+          return;
+        }
+        if (response.status === 'failure' && response.error) {
+          onError(response.error);
+        }
+        onError(new OTLPExporterError('Export failed with unknown error'));
```

Comment:
This code path can call `onError` twice. Should that last `onError` be inside an `} else {` block?

Reply:
Yes, you're right. Good catch.
Fixed in [5e40f0d](https://github.com/open-telemetry/opentelemetry-js/pull/4889/commits/5e40f0dda6d8572acaf5bd63ad16c92d73ae46f5)